tests/Tests.elm

```diff
@@ -46,3 +50,118 @@ readmeExample =
+
+                        step gen =
+                            Random.step gen seed
+
+                        aFuzzer =
+                            tuple5
+                                ( tuple ( list int, array float )
+                                , maybe bool
+                                , result unit char
+                                , tuple4
+                                    ( percentage
+                                    , map2 (+) int int
+                                    , frequencyOrCrash [ ( 1, constant True ), ( 3, constant False ) ]
+                                    , Fuzz.filter (\i -> i % 10 /= 0) int
+                                    )
+                                , tuple3 ( intRange 0 100, floatRange -51 pi, map abs int )
+                                )
+
+                        valNoShrink =
+                            aFuzzer |> Fuzz.Internal.unpackGenVal |> step |> fst
+
+                        valWithShrink =
+                            aFuzzer |> Fuzz.Internal.unpackGenTree |> step |> fst |> RoseTree.root
+                    in
+                        Expect.equal valNoShrink valWithShrink
+            ]
+        ]
+
+
+shrinkingTests : Test
+shrinkingTests =
+    describe "Tests that fail intentionally to test shrinking"
+        [ fuzz2 int int "Every pair of ints has a zero" <|
+            \i j ->
+                (i == 0)
+                    || (j == 0)
+                    |> Expect.true "This should fail with (1,1)"
+        , fuzz3 int int int "Every triple of ints has a zero" <|
+            \i j k ->
+                (i == 0)
+                    || (j == 0)
+                    || (k == 0)
+                    |> Expect.true "This should fail with (1,1,1)"
+        , fuzz4 int int int int "Every 4-tuple of ints has a zero" <|
+            \i j k l ->
+                (i == 0)
+                    || (j == 0)
+                    || (k == 0)
+                    || (l == 0)
+                    |> Expect.true "This should fail with (1,1,1,1)"
+        , fuzz5 int int int int int "Every 5-tuple of ints has a zero" <|
+            \i j k l m ->
+                (i == 0)
+                    || (j == 0)
+                    || (k == 0)
+                    || (l == 0)
+                    || (m == 0)
+                    |> Expect.true "This should fail with (1,1,1,1,1)"
+        , fuzz (list int) "All lists are sorted" <|
+            \aList ->
+                let
+                    checkPair l =
+                        case l of
+                            a :: b :: more ->
+                                if a > b then
+                                    False
+                                else
+                                    checkPair (b :: more)
+
+                            _ ->
+                                True
+                in
+                    checkPair aList |> Expect.true "This should fail with [1,0] or [0,-1]"
+        ]
```

Comment:
Yes, but they can't be run by CI because they fail, intentionally. I've got an idea to solve that but it doesn't block the PR or even the release.